src/attributes/limits.md

```diff
@@ -4,10 +4,12 @@ The following [attributes] affect compile-time limits.
 
 ## The `recursion_limit` attribute
 
-The *`recursion_limit` attribute* may be applied at the crate level to set the
+The *`recursion_limit` attribute* may be applied at the [crate] level to set the
 maximum depth for potentially infinitely-recursive compile-time operations
 like auto-dereference or macro expansion. It uses the [_MetaNameValueStr_]
```

Comment:
```suggestion
like macro expansion or auto-dereference. It uses the [_MetaNameValueStr_]
```
since macro expansion is much more common; I didn't even know auto deref was subject to this.

Reply:
There are many processes which check the limit:

https://github.com/rust-lang/rust/blob/ef4d1c419525e1510aa650e2bec0d8f7449a2130/src/librustc/traits/project.rs#L349-L358
https://github.com/rust-lang/rust/blob/ef4d1c419525e1510aa650e2bec0d8f7449a2130/src/librustc/traits/project.rs#L864-L868
https://github.com/rust-lang/rust/blob/ef4d1c419525e1510aa650e2bec0d8f7449a2130/src/librustc/traits/select.rs#L1121-L1131
https://github.com/rust-lang/rust/blob/ef4d1c419525e1510aa650e2bec0d8f7449a2130/src/librustc/traits/query/normalize.rs#L100-L108
https://github.com/rust-lang/rust/blob/ef4d1c419525e1510aa650e2bec0d8f7449a2130/src/librustc/ty/layout.rs#L168-L174
https://github.com/rust-lang/rust/blob/ef4d1c419525e1510aa650e2bec0d8f7449a2130/src/librustc_mir/monomorphize/collector.rs#L452-L460
https://github.com/rust-lang/rust/blob/ef4d1c419525e1510aa650e2bec0d8f7449a2130/src/librustc_traits/dropck_outlives.rs#L162-L168
https://github.com/rust-lang/rust/blob/ef4d1c419525e1510aa650e2bec0d8f7449a2130/src/librustc_typeck/check/autoderef.rs#L52-L58
https://github.com/rust-lang/rust/blob/ef4d1c419525e1510aa650e2bec0d8f7449a2130/src/libsyntax/ext/expand.rs#L525-L537